src/fragments/lib/storage/native_common/getting-started/common.mdx

```diff
@@ -1,5 +1,9 @@
 The Amplify Storage category provides an interface for managing user content for your app in public, protected, or private storage buckets. The Storage category comes with default built-in support for Amazon Simple Storage Service (S3). The Amplify CLI helps you to create and configure the storage buckets for your app. The Amplify AWS S3 Storage plugin leverages [Amazon S3](https://aws.amazon.com/s3).
 
+import flutter0 from "/src/fragments/lib/storage/flutter/getting-started/50_developerPreview.mdx";
+
+<Fragments fragments={{flutter: flutter0}} />
```

Comment:
I think we need a slightly different fragment since this one actually says Auth. Maybe we can even change that fragment to say we have support for Auth, API, Analytics, and Storage?

Reply:
I update the fragment, and kept it storage only. Thinking this is under the storage category, mentioning only about storage developer preview may be less distraction?